client/js/components/map/publicdetail/DiplanKarteWrapper.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="h-full w-full">
+    <dp-button
+      id="statementModalButton"
+      :class="prefixClass('left-[365px] top-[24px] pt-[11px] pb-[11px] pl-[20px] pr-[20px] absolute z-above-zero')"
+      data-cy="statementModal"
+      :disabled="!hasPermission('feature_new_statement')"
+      href="#publicStatementForm"
+      rounded
+      :text="Translator.trans('statement.participate')"
+      @click.stop.prevent="() => hasPermission('feature_new_statement') ? toggleStatementModal({}) : null" />
+
+    <diplan-karte />
+  </div>
+</template>
+
+
+<script setup>
+import { DpButton, prefixClassMixin } from '@demos-europe/demosplan-ui'
+import { MapPlugin } from '@init/diplan-karten'
+import { getCurrentInstance } from 'vue'
+
+const instance = getCurrentInstance()
+
+instance.appContext.app.mixin(prefixClassMixin)
+instance.appContext.app.use(MapPlugin, {
+  template: {
+    compilerOptions: {
+      isCustomElement: (tag) => tag === "diplan-karte",
+    }
+  }
+})
+
+const toggleStatementModal = (updateStatementPayload) => {
+  console.log('updateStatementPayload', instance.parent)
```

Comment:
please remove this console.log here

Reply:
Thanks, done in 0668a1494e00b9a09d0853d245c396d9dcd9852b